packaging/installer/functions.sh

```diff
@@ -933,7 +933,7 @@ portable_add_group() {
   groupname="${1}"
 
   # Check if group exist
-  if cut -d ':' -f 1 < /etc/group | grep "^${groupname}$" 1> /dev/null 2>&1; then
+  if getent group | grep -q "^${groupname}:"; then
```

Comment:
I see the syntax is 

> getent [option]... database key...

This is nitpicking, but `grep` is not really needed

```bash
if getent group "${groupname}" > /dev/null; then
```

Reply:
This is correct, just calling `getent` with the group name is sufficient to check for the existence of the group.

Reply:
Good point. Done.